march_hardware/src/MarchRobot.cpp

```diff
@@ -14,14 +14,10 @@
 
 namespace march4cpp
 {
-MarchRobot::MarchRobot()
-{
-}
-
 MarchRobot::MarchRobot(::std::vector<Joint> jointList, ::std::string ifName, int ecatCycleTime)
 {
-  this->jointList = jointList;
-  ethercatMaster.reset(new EthercatMaster(&jointList, ifName, this->getMaxSlaveIndex(), ecatCycleTime));
+  this->jointList = std::move(jointList);
```

Comment:
Is this still necessary?

Reply:
Necessary no, but since the old variable is no longer needed we might as well move it instead of copying.